docs/common/help.md

```diff
@@ -0,0 +1,45 @@
+How to Get Help
+===============
+
+This page is aimed at OSG site administrators looking for support. Help for OSG users is found at [our support desk](https://support.opensciencegrid.org/support/home).
+
+Grid Operations Center
+----------------------
+
+The Grid Operations Center (GOC) is available to coordinate users, site admins, and developers around an issue.  Additionally, the GOC can provide basic monitoring and troubleshooting.  There are several ways to receive support:
+
+*  You can [submit a trouble ticket](https://ticket.grid.iu.edu) or send an email to [goc@opensciencegrid.org](mailto:goc@opensciencegrid.org) (which also accept general inquiries not intended for tickets.)
+*  The [trouble ticket system](https://ticket.grid.iu.edu/goc/list/open) is searchable.  Historical tickets may contain the solution for similar problems others have encountered.
+*  The [operations blog](http://osggoc.blogspot.com/) contains information about recent software releases, and important outage and maintenance notifications of central OSG services.
+*  For emergencies, the OSG Grid Operation Center provides extended support. Operators are on hand 24x7 at the GOC and can be reached via phone at +1 317-278-9699.   Non-emergency issues can be opened 24x7 but will be handled during normal business hours.
+
+
+Security incident
+-----------------
+
+Security incidents can be reported by following the instructions on the [Incident Discovery and Reporting](https://opensciencegrid.github.io/security/IncidentDiscoveryReporting/) page.  Additional steps to aid in the incident handling process are also linked from that page.
+
+Information Required to Help You
+--------------------------------
+
+If you came to this page from an installation or other document in this website, then follow instructions in the **Troubleshooting** and **Debugging** sections of the document and include results in your support inquiry, no matter which channel you choose (email, trouble ticket, web chat, ...)
```

Comment:
... then follow instructions in the Troubleshooting and Debugging sections of **that** document and include ...